roofit/roofit/src/RooDSCBShape.cxx

```diff
@@ -7,7 +7,7 @@
  *    Michael Wilkinson add to RooFit source                                 *
  *****************************************************************************/
 
-/** \class RooCBShape
+/** \class RooDSCBShape
```

Comment:
Good catch! Unfortunately, this will clash with https://github.com/root-project/root/pull/7257 which is ready to merge. If you don't mind, I would be happy to merge the crystal ball PR first so we can save one round of jenkins tests (since this PR is `[skip-ci]`).

Reply:
Ok, gogo! :-) @guitargeek